lnwire/neighbor_ack.go

```diff
@@ -10,18 +10,19 @@ import (
 )
 
 type NeighborAckMessage struct {
-	RoutingMessageBase
 }
 
 func (msg *NeighborAckMessage) String() string {
-	return fmt.Sprintf("NeighborAckMessage{%v %v}", msg.SenderID, msg.ReceiverID)
+	return fmt.Sprintf("NeighborAckMessage{}",)
 }
 
 func (msg *NeighborAckMessage) Command() uint32 {
 	return CmdNeighborAckMessage
 }
 
 func (msg *NeighborAckMessage) Encode(w io.Writer, pver uint32) error {
+    // Transmission function work incorrect with empty messages so write some random string to make message not empty
+    w.Write([]byte("NeighborAckMessage"))
```

Comment:
Yes. However,  it is easy to determine message type by looking at message body which may be useful in debugging.